src/api/repos/activity-repo.ts

```diff
@@ -69,10 +69,12 @@ const updateActivity = async (id: number, update: any) => {
         console.log(err);
         throw new BaseError('ORM Sequelize Error', 'There has been an error in updating the Activity', 'updateActivity', httpStatusCode.INTERNAL_SERVER, true);
     });
-    return {
-        success: !!result,
-        activity: result[1][0]
-    };
+    if (returning)
```

Comment:
What is returning? is this to keep track if there is multiple updates on the activity? Why do we need it?

Reply:
that's to indicate if we want the whole object `activity` in the response